nav2_map_server/test/unit/test_map_io_2d.cpp

```diff
@@ -33,19 +33,14 @@
 
 #include <gtest/gtest.h>
 #include <experimental/filesystem>
-#include <stdexcept>
 #include <string>
 #include <vector>
-#include <memory>
-#include <iostream>
-#include <fstream>
```

Comment:
Something strange here: why are you removing so much headers?

Reply:
`stdexcept` not used https://en.cppreference.com/w/cpp/header/stdexcept
`memory` not used https://en.cppreference.com/w/cpp/header/memory
`iostream` not used https://www.geeksforgeeks.org/basic-input-output-c/
`fstream` not used `map_io_2d` deals with file io internally
`yaml.h` previously included in `map_io_2d.hpp`, also not directly used in test.
`map_io.hpp` changed and moved to `map_2d/map_io_2d.hpp`
`map_server.hpp` not used the test case is meant for `2D_io`

Reply:
Good explanation. Thanks!